docker-app/worker_wrapper/wrapper.py

```diff
@@ -431,7 +430,7 @@ def after_docker_run(self) -> None:
                 if package_id in job_ids:
```

Comment:
this can be moved to the if above:

` if package_id ==str(self.job.project.last_package_job_id) or package_id in job_ids:
`

Reply:
This is not a change of this specific PR, is it a must to apply this change?

Also, I would argue that small individual `if`s are much more readable than a bunch of conditions in one place.